web/src/components/organisms/Asset/AssetList/hooks.ts

```diff
@@ -93,17 +95,17 @@ export default (projectId?: string) => {
             });
             if (result.errors || !result.data?.createAsset) {
               // TODO: notification
-              console.log("Failed to add one or more assets.");
+              console.log(t("Failed to add one or more assets."));
             }
           }),
         );
         if (results) {
           // TODO: notification
-          console.log("Successfully added one or more assets.");
+          console.log(t("Successfully added one or more assets."));
```

Comment:
@nourbalaha , remove the logging code `console.log`, will replace it with notifications.

Reply:
Can you explain more please, do you mean that I should remove the console.log line? replace it with what ??
As far as I know, we haven't implemented notifications in reearth-cms, no?

Reply:
Yes, remove `console.log` no need for it here, it's for debugging purposes only.